src/main/java/hudson/tasks/junit/SuiteResult.java

```diff
@@ -348,4 +349,22 @@ void setParent(hudson.tasks.junit.TestResult parent) {
     private static final long serialVersionUID = 1L;
 
     private static final Pattern SUREFIRE_FILENAME = Pattern.compile("TEST-(.+)\\.xml");
+
+    /**
+     * Merges another SuiteResult into this one.
+     * 
+     * @param sr
+     */
+    public void merge(SuiteResult sr) {
+        if (sr.hasTimeAttr ^ hasTimeAttr){
+            throw new IllegalStateException("Merging of suiteresults with incompatible time attribute usage is not supported.( "+getFile()+", "+sr.getFile()+")");
```

Comment:
I've changed this exception into a logger call...since the severity of this is not really critical - and theres no proper way to handle this elsewhere...so it will log a warning in this case.